test/test-pages.js

```diff
@@ -17,27 +17,22 @@ it('Landing page status', function (done) {
 });
 
 it('Dashboard categories page', function (done) {
-  request('http://localhost:3000/dashboard/testorg/categories', function (error, response, body) {
-    expect(response.statusCode).to.equal(200);
-    expect(body).to.contain("testorg's Dashboard");
-    expect(body).to.contain("Add a category");
+  request('http://localhost:3000/dashboard/categories', function (error, response, body) {
```

Comment:
Could you use BASE_URL instead of hardcode localhost:3000?

Same comment to the other test cases.